components/contact/ContactSection.tsx

```diff
@@ -2,12 +2,34 @@ import Markdown from 'markdown-to-jsx'
 import ContactSectionRow from './ContactSectionRow'
 import { programs } from '../../lib/programs'
 
-export default function ContactSection({ title, intro, id, details }) {
+interface Item {
+  content: string
+  id: string
+  icon: string
+  button: boolean
+  highlight?: boolean
+  link: string
+}
+
+interface Detail {
+  id: string
+  color: undefined
```

Comment:
We are using it, it's because the highlight is a boolean and the color is a string. and they are together on the same line. If you have another solution, please let me know.

Reply:
In our mapper, color corresponds to `scBackgroundColour` coming from AEM. The values are either `true`, `false` or `null`, so the type for color should be set as boolean.